README.md

```diff
@@ -21,6 +21,8 @@ Usage
         Google:a2b3c4d5e6f7ghij
         Github:234567qrstuvwxyz
 
```

Comment:
Why the newline? Shouldn't it have a name too?

Reply:
I don't think there's a newline here, that's probably just GitHub wrapping the long line.

We could safely drop the padding bytes on the `secret` to tighten it up.

The `otpauth://` form gets its name from the URL body—in this case `testuser`.

Reply:
Pretty sure line 23 is empty.

Reply:
Oh, sorry, I thought you meant on the example below. 👍🏼

Reply:
Let me remove that empty line.